docs/eni-and-ip-target.md

```diff
@@ -0,0 +1,49 @@
+## `WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET` 
+
+The AWS VPC CNI has an [IPAM](https://en.wikipedia.org/wiki/IP_address_management) daemon (`aws-node`) running on
```

Comment:
well, technically `aws-node` is the name of the Daemonset object in Kubernetes, whereas `aws-k8s-agent` is technically the name of the daemon program that provides the ipamd service :)

Reply:
In other words, I would change "daemon" to be "Kubernetes Daemonset`